stl/inc/xcharconv_ryu.h

```diff
@@ -1114,7 +1114,7 @@ _NODISCARD inline __floating_decimal_32 __f2d(const uint32_t __ieeeMantissa, con
         // __mp = __mv + 2, so it always has at least one trailing 0 bit.
         --__vp;
       }
-    } else if (__q < 31) { // TODO(ulfjack): Use a tighter bound here.

```

Comment:
Similarly, can you change `TODO(ulfjack)` in these comments to `TRANSITION` ? (These comments come from the upstream Ryu project where they’re notes-to-self for the algorithm’s inventor, Ulf Adams.)